src/univariate/continuous/beta.jl

```diff
@@ -116,8 +116,85 @@ gradlogpdf(d::Beta{T}, x::Real) where {T<:Real} =
 
 #### Sampling
 
-rand(d::Beta) = StatsFuns.RFunctions.betarand(d.α, d.β)
+struct BetaSampler{T<:Real, S1 <: Sampleable{Univariate,Continuous},
+                   S2 <: Sampleable{Univariate,Continuous}} <:
+    Sampleable{Univariate,Continuous}
+    γ::Bool
```

Comment:
unicode is not practical in fields people have to type (struct fields and keyword arguments to functions, maybe switch to `gamma`, `ialpha`, `ibeta`

Reply:
why would they have to access these?

Reply:
This isn't my code, but I tend to agree with @simonbyrne - these are just internal to the sampler.

Reply:
ok to leave it as such if not belonging to the PR, but defining new behavior on samplers for example.